pages/DetailsTrip/[city].js

```diff
@@ -0,0 +1,137 @@
+import { useRouter } from "next/router";
+import {
+  StyleButton,
+  StyleList,
+  StyleDiv,
+  StyleCountry,
+  StyleSection,
+  StyleSpan,
+  StyleH4,
+  StyleTotalPrice,
+} from "../../src/components/StyledTripDetails";
+import Header from "@/src/components/Header/Header";
+import { countries } from "@/lib/db";
+
+export default function DetailsOfTrip() {
+  const router = useRouter();
+  const cityName = router.query.city;
+
+  /* Get details about the city*/
```

Comment:
Usually comments like this are not neccessary if your code is written in a semantic way, like using good Variable and Function names